docs/DataModels/DevLakeDomainLayerSchema.md

```diff
@@ -435,6 +435,20 @@ Events of pull requests.
 
 ### Domain 4 - CI/CD(WIP)
 
+#### cicd_scopes
+
+Information about Jenkins Job, GitHub Action or Gitlab CI. 

```

Comment:
This one-line description is a bit too abstract. Maybe it's easier for users to understand if we explain cicd_scopes as a collection of cicd_pipelines and give some examples.

For GitHub: a GitHub repo is converted to a cicd_scope
For Jenkins: <I'm actually not sure the Jenkins entity that corresponds to a cicd_scope>
For GitLab: I assume it's also a GitLab repo?
Any other data source that may produce cicd_scope?

Reply:
fixed, we also have bitbutcket, but I think we don't need to enumerate every plugin?